cap-ci/jobs/deploy-kubecf.tmpl

```diff
@@ -8,13 +8,7 @@
     {{- end }}
   {{- if index .jobs "deploy-stratos" }}
   - get: helm-chart.stratos-chart
-    {{- if not (index .jobs "deploy-k8s") }}
-    trigger: true
```

Comment:
we still need these triggers for CI using pet clusters

Reply:
For that, I have kept the trigger on `s3.kubecf-bundle`. I don't think we ever need pipelines to automatically trigger on new stratos charts/releases.

Reply:
oh we do. cap-release-ci is to test every piece of CAP. Hence, it should get triggered for every new build of its bits. kubecf is not the olny thing this CI is suppose to test. The trigger was added in deliberately.

Reply:
I am still not fully convinced but I am reverting changes and we can handle it separately.